src/processes/antiraidListener.ts

```diff
@@ -16,29 +16,46 @@ export const antiraidListener: ProcessInterface = {
         const mentions = new Map()
         const mutedRole = config.roleIDs.muted
 
-        // TODO: fix staffRole to mention properly, cannot be string
-        //const staffRole = config.roleIDs.staff
-        const staffRole = 662405675107876864
+        // NOTE: Pull from config, same role is achieved
+        const staffRoleId = config.roleIDs.staff
+        // const staffRole = 662405675107876864
+
+        // NOTE: Get role from config
+        const seniorRoleId = config.roleIDs.senior
+
+        const maxMentionedRoles = config.antiRaidRoles
+        const maxMentionedMemebers = config.antiRaidMembers
+        const maxMentionedEverything = maxMentionedRoles + maxMentionedMemebers
+
+        // DEBUG: Debug console log for role comparison
+        // console.log(staffRole, seniorRole)
 
         client.on('message', async (message: Message) => {
             const lastKnownTimestamp = mentions.get(message.author.id) ?? null
 
-            // TODO: escape check if user is senior
-            //message.member.roles
-
-            const {
-                users: { size: mentionedUsers },
-                roles: { size: mentionedRoles }
-            } = message.mentions
-
-            if (
-                mentionedRoles > 1 ||
-                mentionedUsers > 2 ||
-                mentionedUsers + mentionedRoles > 2
-            ) {
-                lastKnownTimestamp
-                    ? checkOffender(message, lastKnownTimestamp)
-                    : addOffenderToList(message)
+            const userRoleIds = message.member.roles.cache.map(r => r.id)
+
+            // Staff role is not used as of now
```

Comment:
Mhh?

Reply:
Get a list of all the users roles.

Reply:
I meant the comment in Line 37 :D I dont think its neccessary anymore.